services/core-web/src/components/admin/MinespaceUserManagement.js

```diff
@@ -54,6 +62,35 @@ export class MinespaceUserManagement extends Component {
     });
   };
 
+  handleCreateUser = (values) => {
+    this.props.createMinespaceUser(values).then(() => {
+      this.refreshUserData();
+    });
+  };
+
+  handleOpenModal = (e, record) => {
+    this.props.openModal({
+      props: {
+        title: "Update User",
+        closeModal: this.props.closeModal,
```

Comment:
Can you update the title to specify what user is being edited? 

`Update User: ${record.email_or_username}`